inst/examples/demo/www/.sass_cache_keys

```diff
@@ -0,0 +1 @@
+da317e5e167a5ea9251c99350dd6a13b
```

Comment:
Please remove this file. Pay attention to CI checks: they failed because this file was added.